### PR TITLE
[CBRD-24411] Modification of CUBRID installers to maintain the cubrid.sh file consistency

### PR DESCRIPTION
**Pull request by kisoo-han**
http://jira.cubrid.org/browse/CBRD-24411

**Purpose**
- keep 1 copy of  cubrid.sh/cubrid.sh in $CUBRID/share/scripts
- modify installer & install.sh to use above files to set CUBRID Environments

**Implementation**
N/A

**Remarks**

### CODE REVIEW

File: cmake/CPack.STGZ_Header.sh.in [hunk 1]
```diff
@@ -245,37 +184,66 @@ if [ -z "$SHELL" ];then
...
+		;;
+	bash)
+		sh_profile=$HOME/.bash_profile
+		;;
+	sh)
+		sh_profile=$HOME/.profile
+		;;
+	csh | tcsh)
+		sh_profile=$HOME/.cshrc
+		;;
```
Comment (mhoh3963) [inline cmake/CPack.STGZ_Header.sh.in:201]: I think that an error handling routine is required for other shell cases.

File: contrib/scripts/cubrid.csh [hunk 2]
```diff
@@ -1,5 +1,3 @@
-
-setenv CUBRID /opt/cubrid
 setenv CUBRID_DATABASES $CUBRID/databases
```
Comment (mhoh3963) [inline contrib/scripts/cubrid.csh:1]: please add a license comment like cubrid.sh.

Comment (mhoh3963) [inline contrib/scripts/cubrid.csh:19]: please add a license comment like cubrid.sh.

File: cmake/CPack.STGZ_Header.sh.in (hunk 1 above)
Comment (kisoo-han) [reply cmake/CPack.STGZ_Header.sh.in:201]: Thank you for your comment.
Fixed it.

File: contrib/scripts/cubrid.csh (hunk 2 above)
Comment (kisoo-han) [reply contrib/scripts/cubrid.csh:19]: Thank you for your comment, License text is added.

**Review by mhoh3963 — APPROVED** [review]

**Review by airnet73 — APPROVED** [review]

**Pull request merged**